fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -128,6 +128,8 @@
             createResource(REPOSITORY_NAMESPACE + "Binary");
     public static final Resource FEDORA_RESOURCE =
             createResource(REPOSITORY_NAMESPACE + "Resource");
+    public static final Resource FEDORA_PAIR_TREE =
+            createResource(REPOSITORY_NAMESPACE + "Pairtree");
```

Comment:
Do we know that this is still a thing?

Reply:
I'm really not sure, maybe you could add an open question to the decisions/questions document?

Reply:
Added it to the Open Questions https://wiki.lyrasis.org/display/FF/Decisions+and+Open+Questions